### PR TITLE
[INLONG-2650][Feature][Sort] Define sort stand-alone CLS context and config bean

### LINKED ISSUE #2650: [Feature][Sort] Define sort stand-alone CLS context and config bean
### Description

parent issue #2576 

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by imvan**
### Title Name: [INLONG-2650][Feature][Sort] Define sort stand-alone CLS context and config bean

Fixes #2650 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by wardlican — APPROVED** [review]

File: inlong-sort-standalone/pom.xml [hunk 1]
```diff
@@ -59,6 +59,7 @@
         <mockito.version>2.23.0</mockito.version>
         <jakarta.validation.version>2.0.2</jakarta.validation.version>
         <elasticsearch-version>6.8.13</elasticsearch-version>
+        <cls-version>1.0.5</cls-version>
```
Comment (wardlican) [inline inlong-sort-standalone/pom.xml:62]: The same style “xxx.version” is recommended

File: inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSinkContext.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            return;
+        }
+        AsyncProducerConfig producerConfig = new AsyncProducerConfig(
+                idConfig.getEndpoint(),
+                idConfig.getSecretId(),
+                idConfig.getSecretKey(),
+                NetworkUtils.getLocalMachineIP());
+        // todo set other configs
+        AsyncProducerClient client = new AsyncProducerClient(producerConfig);
+        clientMap.put(uid, client);
```
Comment (luchunliang) [inline inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSinkContext.java:162]: It is better to create AsyncProducerClient by endpoint, not to create AsyncProducerClient by id.

**Review by luchunliang — APPROVED** [review]

File: inlong-sort-standalone/pom.xml (hunk 1 above)
Comment (imvan) [reply inlong-sort-standalone/pom.xml:62]: thx, fixed

File: inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSinkContext.java (hunk 2 above)
Comment (imvan) [reply inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSinkContext.java:162]: The init process of AsyncProducerClient requires two important parameters, secretId and secretKey. 
Each id has unique secretId and secretKey but may share the same endpoint. 
Hence we have to new one client for each task, although it cost more resouces, it's the price that we hand over the controll of dispatch and fail retry to CLS SDK.

Comment (imvan) [reply inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSinkContext.java:162]: details can be seen at  https://github.com/TencentCloud/tencentcloud-cls-sdk-java

**Pull request merged**